tests/test_transforms/test_decomposition/test_change_points_segmentation_transform.py

```diff
@@ -0,0 +1,160 @@
+import numpy as np
```

Comment:
Sometimes out defines the constants separately and sometimes not, try do it consistently. Also if you use the same constant like out_column in all of the tests you can define it globally,